app/repositories/scores.py

```diff
@@ -182,26 +237,28 @@ async def fetch_many(
         params["page_size"] = page_size
         params["offset"] = (page - 1) * page_size
 
-    recs = await app.state.services.database.fetch_all(query, params)
-    return [dict(rec) for rec in recs]
+    scores = await app.state.services.database.fetch_all(query, params)
+    return cast(list[Score], scores) if scores is not None else None
```

Comment:
return type is wrong on this func, should be `list[Score]`, but github is too bad to let me edit unchanged code